config/environments/production.rb

```diff
@@ -58,6 +58,12 @@
 
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
+  config.cache_store = :redis_cache_store, {
+    url: ENV["REDIS_URL"],
+    error_handler: -> (method: , returning: exception:) {
```

Comment:
Should there be a comma between returning and exception here?

Reply:
I thought rails had a default cache store built in?

Reply:
> Should there be a comma between returning and exception here?

Good catch! Fixing up.



> I thought rails had a default cache store built in?

[Default cache store for Rails is a Memory Store](https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-memorystore), which means that it lasts for the duration of the Ruby process. Since a rake task runs and exits, it doesn't persist between runs.